js/base-weather.js

```diff
@@ -32,6 +32,7 @@ function baseWeather () {
 			"Waves": new Image(),
 			"Ripples": new Image(),
 			"Blood Rain": new Image(),
+			"Custom": new Image(),
```

Comment:
what's the reasoning behind adding the `Custom` image here/removing the "cleanup when unused" step below..?